overlord/servicestate/quota_control.go

```diff
@@ -82,10 +82,25 @@ func quotaGroupsAvailable(st *state.State) error {
 	return nil
 }
 
+// QuotaGroupCreate reflects all of options available when creating new quota
+// groups.
+type QuotaGroupCreate struct {
```

Comment:
Since these are parameters, maybe the name should reflect that? Something like:
```suggestion
type QuotaGroupCreateOptions struct {
```

Reply:
Maybe a task for a followup PR to rename both, so we remain consistent in the file

Reply:
this is a bit tricky because Update is also a name, but Create is not. So consistent name would have been QuotaGroupCreation, but it's a odd name. In other places we tend to use FuncNameOptions or FuncNameParams

Reply:
I'll just do the rename for both here :)